src/south/OPCUA/OPCUA.schema.jsx

```diff
@@ -45,27 +45,43 @@ schema.form = {
   retryInterval: {
     type: 'OIbInteger',
     newRow: true,
-    md: 3,
+    md: 2,
     valid: minValue(1000),
     defaultValue: 10000,
     help: <div>Retry Interval (ms)</div>,
   },
   maxReadInterval: {
     type: 'OIbInteger',
     newRow: false,
-    md: 3,
+    md: 2,
     valid: minValue(60),
     defaultValue: 3600,
     help: <div>Max read interval (s)</div>,
   },
   readIntervalDelay: {
     type: 'OIbInteger',
     newRow: false,
-    md: 3,
+    md: 2,
     valid: minValue(0),
     defaultValue: 200,
     help: <div>Time to wait between the read interval iterations (ms)</div>,
   },
+  maxReturnValues: {
+    type: 'OIbInteger',
+    newRow: false,
+    md: 2,
+    valid: minValue(0),
+    defaultValue: 1000,
+    help: <div>Max return values</div>,
```

Comment:
Not really useful since it's the same than the field name. I would put something like : 'Max number of values returned for one point during a read interval'